packages/frontend/src/utils/cookies.ts

```diff
@@ -40,7 +41,11 @@ export const setCookie = (cookieName: string, identifier: string) => {
 
     if(consent){
         initializeAmplitude()
+        track(EVENT_NAME.COOKIE_ACCEPTED)
     }else {
         removeCookies()
+        track(EVENT_NAME.COOKIE_REJECTED)
```

Comment:
Will cont work right? As we did not initialiseAmplitude

Reply:
Changed to fathom